doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -0,0 +1,136 @@
+.. _deploy-cephadm-mgmt-gateway:
+
+=============
+Mgmt Gateway
+=============
+
+Deploying mgmt-gateway
+======================
+
+Starting from version 19.x.x, the `mgmt-gateway` service introduces a new design for Ceph applications
+based on a modular, service-based architecture. This service, managed by cephadm and built on top of nginx
+(an open-source, high-performance web server), acts as the new front-end and single entry point to the
+Ceph cluster. The `mgmt-gateway` provides unified access to all Ceph applications, including the ceph dashboard
```

Comment:
Ceph dashboard

I love the pretty diagram in the PR description, though the inclusion of Prometheus there leads me to ask if any service-specific steps are taken.  Prom HA can be tricky if the timestamps aren't deduplicated.

Reply:
@anthonyeleven the first diagram is in fact to describe the legacy architecture (not the new one). That why I didn't include it as part of the docs. Probably I'll be making some nice diagram that describes the new one and I'll include it later on a separate PR :)